accumulo/src/main/scala/geotrellis/spark/io/accumulo/AccumuloLayerHeader.scala

```diff
@@ -35,16 +36,26 @@ object AccumuloLayerHeader {
         "format" -> JsString(md.format),
         "keyClass" -> JsString(md.keyClass),
         "valueClass" -> JsString(md.valueClass),
-        "tileTable" -> JsString(md.tileTable)
+        "tileTable" -> JsString(md.tileTable),
+        "layerType" -> JsString(md.layerType.toString)
       )
 
     def read(value: JsValue): AccumuloLayerHeader =
-      value.asJsObject.getFields("keyClass", "valueClass", "tileTable") match {
+      value.asJsObject.getFields("keyClass", "valueClass", "tileTable", "layerType") match {
+        case Seq(JsString(keyClass), JsString(valueClass), JsString(tileTable), JsString(layerType)) =>
+          AccumuloLayerHeader(
+            keyClass,
+            valueClass,
+            tileTable,
+            LayerType.fromString(layerType)
```

Comment:
There is still no `LayerType` codec?

Reply:
There is https://github.com/jbouffard/geotrellis/blob/36aa9ed1e56e3620dc59a59d6f3e6c77c7e94212/spark/src/main/scala/geotrellis/spark/io/LayerType.scala#L19

Reply:
Ah right, so you can use it here btw :D

Reply:
Oh, that's right. My bad